.claude/skills/cannbot-skills/ascendc-kernel-develop-workflow/scripts/verify_cmake_config.py

```
@@ -0,0 +1,139 @@
+    if not re.search(r'project\s*\([^)]*LANGUAGES[^)]*ASC', content, re.IGNORECASE):
+        errors.append("❌ project() 必须包含 LANGUAGES ASC CXX（当前缺少 ASC）")
+    
+    # 检查3：禁止使用 asc_add_ops_executable（不存在的函数）
+    if 'asc_add_ops_executable' in content:
+        errors.append("❌ 禁止使用 asc_add_ops_executable（不存在的函数，请使用 add_executable）")
+    
+    # 检查4：必须链接 tiling_api
+    if 'tiling_api' not in content:
+        errors.append("❌ 必须链接 tiling_api 库")
+    
+    # 检查5：必须链接 register
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)